src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +119,205 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+		/// This enables scenarios such as reducing the height of the flyout so that it doesn't obscure the tab bar.
+		/// </summary>
+		/// <param name="obj">The object that modify the height of the flyout.</param>
+		/// <param name="value">Defines the height of the flyout.</param>
+		public static void SetFlyoutHeight(BindableObject obj, double value) => obj.SetValue(FlyoutHeightProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetNavBarIsVisible']/Docs/*" />
+		/// <summary>
+		/// Gets if the navigation bar is visible when a page is presented. 
+		/// </summary>
+		/// <param name="obj">The object that modify the navigation bar visibility.</param>
+		/// <returns>true if the navigation bar is visible; otherwise, false.</returns>
 		public static bool GetNavBarIsVisible(BindableObject obj) => (bool)obj.GetValue(NavBarIsVisibleProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetNavBarIsVisible']/Docs/*" />
+
+		/// <summary>
+		/// Controls if the navigation bar is visible when a page is presented. 
+		/// By default the value of the property is <see langword="true"/>.
+		/// </summary>
+		/// <param name="obj">The object that modify the navigation bar visibility.</param>
+		/// <param name="value">true to set the navigation bar as visible; otherwise, false.</param>
 		public static void SetNavBarIsVisible(BindableObject obj, bool value) => obj.SetValue(NavBarIsVisibleProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetNavBarHasShadow']/Docs/*" />
+		/// <summary>
+		/// Gets if the navigation bar has a shadow. 
+		/// </summary>
+		/// <param name="obj">The object that modify if the navigation bar has a shadow.</param>
+		/// <returns>Returns if the navigation bar has a shadow.</returns>
 		public static bool GetNavBarHasShadow(BindableObject obj) => (bool)obj.GetValue(NavBarHasShadowProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetNavBarHasShadow']/Docs/*" />
+
+		/// <summary>
+		/// Controls whether the navigation bar has a shadow. 
+		/// By default the value of the property is <see langword="true"/> on Android, and <see langword="false"/> on other platforms.
+		/// </summary>
+		/// <param name="obj">The object that modify if the navigation bar has a shadow.</param>
+		/// <param name="value">Manages if the navigation bar has a shadow.</param>
 		public static void SetNavBarHasShadow(BindableObject obj, bool value) => obj.SetValue(NavBarHasShadowProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetSearchHandler']/Docs/*" />
+		/// <summary>
+		/// Gets the integrated search functionality in Shell.
```

Comment:
I don't think that this is necessary to self reference Shell here because this is a method within the class, so its implied that it gets the search functionality. Open for suggestions here though since this is a static method. @davidbritch  thoughts here?

Reply:
This comment applies to several methods in this section

Reply:
Removed.